src/homework7/3/tests.py

```diff
@@ -0,0 +1,30 @@
+from contextlib import redirect_stdout
+from io import StringIO
+import sys
+from task_1_3 import add_to_list_in_dict
+import unittest
+
+sys.path.insert(1, "../1")
+
+
+class TestRunner(unittest.TestCase):
+    def test_run(self):
```

Comment:
usually we use different tests for different cases. just a tip you can use them via ddt library
also they can be just splitted.